arviz/stats/stats.py

```diff
@@ -12,7 +12,7 @@
 
 from ..data import convert_to_inference_data, convert_to_dataset
 from .diagnostics import _multichain_statistics, _mc_error, ess
-from .stats_utils import make_ufunc as _make_ufunc, logsumexp as _logsumexp
+from .stats_utils import make_ufunc as _make_ufunc, logsumexp as _logsumexp, ELPDData
```

Comment:
~let's keep everything lowercase~ is this a class?

Reply:
Yes, it is a children class of `pd.Series` with the `__str__` and `__repr__` methods overwritten. I could create a custom class from zero too if there are no strong reasons in favor of using a Series.

Reply:
Sounds fine with the `Series` backend.